app/gui2/src/components/GraphEditor.vue

```diff
@@ -225,6 +233,44 @@ function getNodeContent(id: ExprId): string {
   return node.rootSpan.repr()
 }
 
+/// The content of a node that is copied to the clipboard. It contains the expression and the
+/// metadata. Used for serializing and deserializing the node information.
+interface CopiedNode {
+  expression: string
+  metadata: NodeMetadata | undefined
+}
+
+/// Copy the content of the selected node to the clipboard.
+function copyNodeContent() {
+  const id = nodeSelection.selected.values().next().value
+  const node = graphStore.nodes.get(id)
+  if (node == null) return
+  const content = node.rootSpan.repr()
+  const metadata = projectStore.module?.getNodeMetadata(id)
+  const copiedNode: CopiedNode = { expression: content, metadata }
```

Comment:
The stored format assumes a single node. It is fine to support copying only one for now, but let's make some room for easily extending it later, such as making the top-level object contain an array of nodes already (e.g. `{ nodes: CopiedNode[] }`).